src/pages/pt-br/concepts/islands.md

```diff
@@ -0,0 +1,68 @@
+</IslandsDiagram>
+
+No Astro, você pode utilizar qualquer framework UI suportado (React, Svelte, Vue, etc.) para renderizar ilhas no navegador. Você pode misturar diferentes frameworks em uma mesma página, ou apenas utilizar seu favorito.
+
+A técnica que este padrão arquitetural se baseia é conhecido como **hidratação parcial** ou **seletiva**. Astro se aproveita desta técnica nos bastidores, viabilizando suas ilhas automaticamente.
+
+## Como Ilhas Funcionam no Astro?
+
+**Astro gera todos os websites com zero Javascript no lado do cliente, por padrão.** Utilize um componente de UI frontend construído com [React](https://reactjs.org/), [Preact](https://preactjs.com/), [Svelte](https://svelte.dev/), [Vue](https://vuejs.org/), [SolidJS](https://www.solidjs.com/), [AlpineJS](https://alpinejs.dev/) ou [Lit](https://lit.dev/) e Astro irá automaticamente renderizá-lo como HTML com antecedência e então remover todo o JavaScript. Isso mantém todos os sites rápidos por padrão através da remoção de todo o JavaScript inutilizado da página. 
+
+```astro title="src/pages/index.astro"
+---
+// Exemplo: Utilize um componente React estático na página, sem JavaScript.
+import MeuComponenteReact from '../components/MeuComponenteReact.jsx';
+---
+<!-- 100% HTML, Zero JavaScript carregado na página! -->
+<MeuComponenteReact />
+```
+
+Porém as vezes, JavaScript no lado do cliente é necessário para criar uma UI interativa. Ao invés de forçar sua página inteira a se tornar uma aplicação JavaScript estilo SPA, Astro te pede para criar uma ilha.
+
+```astro title="src/pages/index.astro" ins="client:load"
+---
+// Exemplo: Utilize um componente React dinâmico na página.
+import MeuComponenteReact from '../components/MeuComponenteReact.jsx';
+---
+<!-- Este componente agora é interativo na página! 
+     O resto do seu website continua estático e com zero JS. -->
+<MeuComponenteReact client:load />
+```
+
+Com Ilhas Astro, a maior parte do seu site continua como leve e puro HTML e CSS. No exemplo acima, você apenas adicionou uma única e isolada **ilha de interatividade** sem precisar mudar o resto da sua página.
+
+## Quais são os benefícios das Ilhas?
+
+O mais óbvio benefício de se construir com Ilhas Astro é performance: a maior parte do seu website é convertido em rápido HTML estático e JavaScript é apenas carregado para os componentes individuais que o precisam. JavaScript é um dos assets mais lentos que você pode carregar por byte, então cada byte conta.
+
+Outro benefício é carregamento paralelo, Na ilustração de exemplo acima, a ilha de baixa prioridade "carrossel de imagens" não precisa bloquear a ilha de alta prioridade "cabeçalho". Os dois são carregados em paralelo e hidratados de forma isolada, o que significa que o cabeçalho se torna interativo imediatamente sem precisar esperar que o carrossel, mais pesado, desacelere o carregamento da página.
```

Comment:
```suggestion
Outro benefício é carregamento paralelo, na ilustração de exemplo acima, a ilha de baixa prioridade "carrossel de imagens" não precisa bloquear a ilha de alta prioridade "cabeçalho". Os dois são carregados em paralelo e hidratados de forma isolada, o que significa que o cabeçalho se torna interativo imediatamente sem precisar esperar que o carrossel, mais pesado, desacelere o carregamento da página.
```

Reply:
Maybe thi suggestion would be bettet
```suggestion
Outro benefício é carregamento paralelo. Na ilustração de exemplo acima, a ilha de baixa prioridade "carrossel de imagens" não precisa bloquear a ilha de alta prioridade "cabeçalho". Os dois são carregados em paralelo e hidratados de forma isolada, o que significa que o cabeçalho se torna interativo imediatamente sem precisar esperar que o carrossel, mais pesado, desacelere o carregamento da página.
```

Reply:
I'm going with the second suggestion, thank you!